src/DotNet.Testcontainers/Networks/Configurations/NetworkDriver.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace DotNet.Testcontainers.Networks.Configurations
+{
+  using JetBrains.Annotations;
+
+  /// <summary>
+  ///
+  /// </summary>
+  public readonly struct NetworkDriver
+  {
+    /// <summary>
+    ///
+    /// </summary>
+    public static readonly NetworkDriver Bridge = new NetworkDriver("bridge");
```

Comment:
I wonder if we should add any other drivers like `overlay`, `host`, `macvlan`, or `none`.

Reply:
Yep, I'll add them too.

Reply:
I've added `host` too. I think the other drivers require a bit more configuration.